docs/platforms-release-process.md

```diff
@@ -86,30 +157,46 @@ Ensure all dependencies and subdependencies have Apache-compatible licenses.
     coho check-license -r android
 
 ## Prepare Release
-Increase the version within package.json using SemVer, and remove the `-dev` suffix.
+
+### Remove the `-dev` suffix from version
+
+This command removes `-dev` from the `version` entry in `package.json`:
 
     for l in cordova-android; do ( cd $l; v="$(grep '"version"' package.json | cut -d'"' -f4)"; if [[ $v = *-dev ]]; then v2="${v%-dev}"; echo "$l: Setting version to $v2"; sed -i '' -E 's/version":.*/version": "'$v2'",/' package.json; fi) ; done
 
-In `cordova-android`, also remember to bump the version in `framework/build.gradle`.
+Note: This command [doesn't actually work](https://issues.apache.org/jira/browse/CB-13809). You can also replace `-dev` manually of course.
 
-If the changes merit it, manually bump the major / minor/ patch version in `package.json`. View the changes via:
+Note: In `cordova-android`, also remember to handle the version in `framework/build.gradle`.
```

Comment:
169 can actually be removed. `coho prepare-release-branch` command that is run later will update build.gradle

Reply:
Is this good enough? I was somehow thinking there was a reason that "remove -dev" and the other version stuff later is split in 2 parts?

Reply:
The remove dev part is split from the other version stuff because we use the version you set in package.json when writing to releasenotes. After you remove dev you may manually change the versionwhen reviewing commits in case you want a patch/minor/major release. This happens right before we run the update releasenotes command

Reply:
Ah, now I finally got it I think. Updated the file to document this in more detail: https://github.com/apache/cordova-coho/pull/170/commits/efe62e3d9d56b6458bc323214926c4efa747c175